integration_tests/run_pyspark_from_build.sh

```diff
@@ -197,11 +197,12 @@ else
       "$LOCAL_ROOTDIR"/src/main/python)
 
     REPORT_CHARS=${REPORT_CHARS:="fE"} # default as (f)ailed, (E)rror
+    PYTEST_COLOR=${PYTEST_COLOR:-yes}
```

Comment:
Nit: Change seems unrelated unless there was an error of omission and somewhere else was supposed to override this environment variable to turn of colors?

Reply:
it should not change anything for existing uses because "yes" is currently hard-coded but I agree it is not related to the main intent of this PR